docs/data/date-pickers/validation/validation.md

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
Here is a diff for the remaining punctuation suggestions 🙈 
```diff
diff --git a/docs/data/date-pickers/validation/validation.md b/docs/data/date-pickers/validation/validation.md
index dad257c9..06a72ae0 100644
--- a/docs/data/date-pickers/validation/validation.md
+++ b/docs/data/date-pickers/validation/validation.md
@@ -13,7 +13,7 @@ By default they provide visual feedback if the component value doesn't meet the
 The validation props are showcased for each type of picker component (`TimePicker`, `DatePicker`, `DateRangePicker`, etc).
 But the same props are available on all component variants.
 
-For example, the validation props available on the `DatePicker` component are also available on:
+For example—the validation props available on the `DatePicker` component are also available on:
 
 - `DesktopDatePicker`
 - `MobileDatePicker`
@@ -71,7 +71,7 @@ The `maxDate` prop prevents the selection of all values after `props.maxDate`.
 
 The `shouldDisableDate` prop prevents the selection of all dates for which it returns `true`.
 
-In the example below, the weekends are not selectable:
+In the example below—the weekends are not selectable:
 
 {{"demo": "DateValidationShouldDisableDate.js", "defaultCodeOpen": false}}
 
@@ -79,15 +79,15 @@ In the example below, the weekends are not selectable:
 `shouldDisableDate` only prevents the selection of disabled dates on the `day` view.
 For performance reasons—when rendering the `month` view—we are not calling the callback for every day of each month to see which one should be disabled (same for the `year` view).
 
-If you know that all days of some months are disabled, you can provide the [`shouldDisableMonth`](#disable-specific-months) prop to disable them in the `month` view.
+If you know that all days of some months are disabled—you can provide the [`shouldDisableMonth`](#disable-specific-months) prop to disable them in the `month` view.
 Same with the [`shouldDisableYear`](#disable-specific-years) prop for the `year` view.
 :::
 
 #### Disable specific dates in range components [<span class="pro-premium"></span>](/x/introduction/licensing/#pro-plan)
 
-For components supporting date range edition (`DateRangePicker`, `DateTimeRangePicker` 🚧), the `shouldDisableDate` prop receives a second argument to differentiate the start and the 
end date.
+For components supporting date range edition (`DateRangePicker`, `DateTimeRangePicker` 🚧)—the `shouldDisableDate` prop receives a second argument to differentiate the start and the e
nd date.
 
-In the example below, the start date can't be in the weekend but the end date can.
+In the example below—the start date can't be in the weekend but the end date can.
 
 {{"demo": "DateRangeValidationShouldDisableDate.js", "defaultCodeOpen": false}}
 
@@ -101,7 +101,7 @@ The `shouldDisableMonth` prop prevents the selection of all dates in months for
 `shouldDisableMonth` only prevents the selection of disabled months on the `day` and `month` views.
 For performance reasons—when rendering the `year` view—we are not calling the callback for every month of each year to see which one should be disabled.
 
-If you know that all months of some years are disabled, you can provide the [`shouldDisableYear`](#disable-specific-years) prop to disable them in the `year` view.
+If you know that all months of some years are disabled—you can provide the [`shouldDisableYear`](#disable-specific-years) prop to disable them in the `year` view.
 :::
 
 ### Disable specific years
@@ -146,7 +146,7 @@ shouldDisableTime={(timeValue, view) => view === 'minutes' && timeValue >= 45};
 shouldDisableTime={(timeValue, view) => view === 'seconds' && timeValue >= 30};
 ```
 
-In the example below, the last quarter of each hour is not selectable.
+In the example below—the last quarter of each hour is not selectable.
 
 {{"demo": "TimeValidationShouldDisableTime.js", "defaultCodeOpen": false}}
 
@@ -163,7 +163,7 @@ The `maxDateTime` prop prevents the selection of all values after `props.maxDate
 {{"demo": "DateTimeValidationMaxDateTime.js", "defaultCodeOpen": false}}
 
 :::warning
-If you want to put time boundaries independent of the date, use the [`time boundaries`](#set-time-boundaries) instead.
+If you want to put time boundaries independent of the date—use the [`time boundaries`](#set-time-boundaries) instead.
 
 ```tsx
 // Disable the values between 6 PM and midnight for every day (tomorrow 5 PM is not disabled).
@@ -183,9 +183,9 @@ If you want to put time boundaries independent of the date, use the [`time bound
 
 ## Show the error
 
-To render the current error, you can subscribe to the `onError` callback which is called every time the error changes.
+To render the current error—you can subscribe to the `onError` callback which is called every time the error changes.
 You can then use the `helperText` prop of the `TextField` to pass your error message to your input as shown below.
 
-Try to type a date that is inside the first quarter of 2022, the error will go away.
+Try to type a date that is inside the first quarter of 2022—the error will go away.
 
 {{"demo": "RenderErrorUnderField.js"}}
```